packages/shared/src/variants/baduk.ts

```diff
@@ -190,19 +216,48 @@ export class Baduk extends AbstractGame<BadukConfig, BadukState> {
     this.phase = "gameover";
   }
 
-  defaultConfig(): BadukConfig {
+  defaultConfig(): GridBadukConfig {
     return { width: 19, height: 19, komi: 6.5 };
   }
 }
 
 /** Returns true if the group containing (x, y) has at least one liberty. */
-export function groupHasLiberties(group: CoordinateLike[], board: Grid<Color>) {
+export function groupHasLiberties(
+  group: CoordinateLike[],
+  board: BadukBoard<Color>,
+) {
   const outer_border = getOuterBorder(group, board);
   const border_colors = outer_border.map((pos) => board.at(pos));
   return border_colors.includes(Color.EMPTY);
 }
 
 /** Returns a reducer that will count occurences of a given number **/
+// This was used before... will need to figure out where it went
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
```suggestion
```

I added these comments so suppress the linter.  Should be good to remove now that we reverted the `reduce(count_color())` code

Reply:
removed comments